src/components/CurveSwitch/CurveSwitch.vue

```diff
@@ -53,10 +53,10 @@ interface IProps {
 
 const props = defineProps<IProps>();
 
-const toast = useToast();
 const { curves } = useStore();
 const store = useSwapStore();
 const tokenStore = useTokensStore();
+// const toast = useToast();
```

Comment:
Why is it commented here, but still in use in code below?

Reply:
Yeah, wrong context assignment in main.ts file leed to incorrect working of ToastService. 

fixed